composer.json

```diff
@@ -16,7 +16,7 @@
   ],
   "autoload": {
     "psr-4": {
-      "Bugo\\FontAwesomeHelper\\": "src/FontAwesomeHelper/"
+      "Bugo\\FontAwesomeHelper\\": "src/"
```

Comment:
I probably would have just named that FAHelper. But I hate mile long namespaces :P